src/entities/Proposal/routes.ts

```diff
@@ -1,57 +1,58 @@
-import { Request } from 'express'
-import { v1 as uuid } from 'uuid'
-import { AlchemyProvider, Block } from '@ethersproject/providers'
-import routes from "decentraland-gatsby/dist/entities/Route/routes";
-import { auth, WithAuth } from "decentraland-gatsby/dist/entities/Auth/middleware";
-import handleAPI, { handleJSON } from 'decentraland-gatsby/dist/entities/Route/handle';
-import validate from 'decentraland-gatsby/dist/entities/Route/validate';
-import schema from 'decentraland-gatsby/dist/entities/Schema'
-import { SNAPSHOT_SPACE, SNAPSHOT_ACCOUNT, SNAPSHOT_ADDRESS, SNAPSHOT_DURATION, signMessage } from '../Snapshot/utils';
-import { Snapshot, SnapshotResult, SnapshotSpace, SnapshotStatus } from '../../api/Snapshot';
-import { Discourse, DiscoursePost } from '../../api/Discourse';
-import { DISCOURSE_AUTH, DISCOURSE_CATEGORY } from '../Discourse/utils';
-import Time from 'decentraland-gatsby/dist/utils/date/Time';
-import ProposalModel from './model';
+import { Request } from "express"
```

Comment:
I'm noticing mi IDE config reorganized importas and changed single quotes for double quotes. I'll change this config to match our codebase